mypy/myunit/__main__.py

```diff
@@ -6,4 +6,13 @@
 
 from mypy.myunit import main
 
+# In Python 3.3, mypy.__path__ contains a relative path to the mypy module
+# (whereas in later Python versions it contains an absolute path).  Because the
+# test runner changes directories, this breaks non-toplevel mypy imports.  We
+# fix that problem by fixing up the path to be absolute here.
```

Comment:
Yeah, I'd prefer to get rid of this hack too.  Interestingly, the fast parser is _not_ the only place where we have a delayed import!  We also do it in [subtypes](https://github.com/ddfisher/mypy/blob/8e7db19fb05659e2c4d40072dd3e39b1a5a88471/mypy/subtypes.py#L292) and [strconv](https://github.com/ddfisher/mypy/blob/8e7db19fb05659e2c4d40072dd3e39b1a5a88471/mypy/strconv.py#L405).  I guess we just don't hit those codepaths in testing?  Yikes...